docker-compose.yaml

```diff
@@ -0,0 +1,44 @@
+# 网络配置
+networks:
+  devNetwork:
+    driver: ${NETWORKS_DRIVER}
+
+# 服务容器配置
+services:
+  mysql:
+    image: mysql:8.0
+    privileged: true
+    environment:
+      - TZ=${TZ}
+      - MYSQL_USER=${MYSQL_USERNAME}                  # 设置 Mysql 用户名称
```

Comment:
How do we supply these variables, should we put them into a `.env` file?

Reply:
The .env file is ignored, should I cancel this restriction?